ror_test/app/controllers/colleges_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CollegesController < ApplicationController
+    def new
+        @college = College.new
+    end
+
+    def index
+        if params[:search]
+            @colleges = College.where('name LIKE ?', "%#{params[:search]}%")
+        else
+            @colleges = College.all
```

Comment:
here in @colleges we have to store all the columns like name ,extended year 
and in college table we have only these 2 columns